_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp

```diff
@@ -311,7 +311,7 @@ class GetDefault
         , mfxU16(*pBL1)[8])
     {
         bool bExternal = false;
-        mfxU16 maxFwd, maxBwd;
+        mfxU16 maxFwd = 0, maxBwd = 0;
```

Comment:
Since these are initialized by the std::tie function, I don't think you need to explicitly initialize them, but you can.  I would call this a KW false positive.